lib/Demangling/Remangler.cpp

```diff
@@ -2111,9 +2111,10 @@ ManglingError Remangler::mangleImplFunctionType(Node *node, unsigned depth) {
       }
       case Node::Kind::ImplFunctionAttribute: {
         char FuncAttr = llvm::StringSwitch<char>(Child->getText())
-                        .Case("@Sendable", 'h')
-                        .Case("@async", 'H')
-                        .Default(0);
+                            .Case("@Sendable", 'h')
+                            .Case("@async", 'H')
+                            .Case("sending-result", 'T')
```

Comment:
The new demangler does not produce an `ImplFunctionAttribute` for this; it produces `ImplSendingResult`.